### PR TITLE
Added ST_Distance_Sphere and ST_Distance to mysql

### PR DESCRIPTION
**Pull request by 0x6368**

### CODE REVIEW

**Comment by sela** [comment]
when will this be merged?

**Comment by holtkamp** [comment]
Note sure this is still on the radar for @sadortun ...

**Comment by sela** [comment]
> Note sure this is still on the radar for @sadortun ...

well i just added the ST_Distance_Sphere internally as needed it calculate distances for my project.

**Comment by asikrhmn** [comment]
Any plan to merge this?

**Review by albertwitrac — APPROVED** [review]

**Review by Mathieu33260 — APPROVED** [review]

**Review by mtexeiraultra — APPROVED** [review]

File: phpunit.xml.dist [hunk 1]
```diff
@@ -14,12 +14,12 @@
...
 
     <php>
-        <var name="db_type" value="pdo_pgsql"/>
-        <var name="db_host" value="localhost" />
-        <var name="db_username" value="postgres" />
-        <var name="db_password" value="" />
+        <var name="db_type" value="pdo_mysql"/>
+        <var name="db_host" value="192.168.99.100" />
+        <var name="db_username" value="root" />
+        <var name="db_password" value="fdklgjhsaljf" />
```
Comment (sadortun) [inline phpunit.xml.dist:20]: Not sure this should be modified 🥲

**Review by sadortun — CHANGES_REQUESTED** [review]
@0x6368 looks good except DB credentials that are now set to some random thing.

**Comment by sadortun** [comment]
Thanks!

**Pull request merged**